src/inc-lib-init.php

```diff
@@ -1084,336 +1084,525 @@ function lovd_getInstallURL ($bFull = true)
+            return false;
+        }
+        foreach (array('qter', 'pter', 'cen', '::', 'sup', 'bsr', 'rer') as $sUnsupported) {
+            if (strpos($sVariant, $sUnsupported)) {
+                $aResponse['errors']['ENOTSUPPORTED'] =
+                    'Currently, "' . $sUnsupported . '" is not yet supported by this HGVS check.';
+                return $aResponse;
+            }
+        }
+        return false;
+    }
+
+
+    // Storing the variant type.
+    if (!$aVariant['type']) {
+        // If no type is given, we can be sure that the variant is either a full wildtype or a full
+        //  unknown variant; so either g.= or g.? . In this case, we do not need to go over632 all tests.
+        if (in_array($aVariant['prefix'], array('c', 'n'))) {
+            $aResponse['position_end_intron'] = 0;
+            $aResponse['position_start_intron'] = 0;
+        }
+        $aResponse['type'] = (substr($aVariant['complete'], -1) == '='? '=' : NULL);
+        
+        if ($aResponse['type'] == '=') {
+            $aResponse['errors']['EMISSINGPOSITIONS'] =
+                'When using "=", always provide the position(s).';
+            return ($bCheckHGVS? false : $aResponse);
+        }
+        return ($bCheckHGVS? true : $aResponse);
+        
+        
+    } elseif (strpos($aVariant['type'], '=') !== false) {
+        if (substr_count($sVariant, '/') == 1) {
+            $aResponse['type'] = 'mosaic';
+        } elseif (substr_count($sVariant, '/') == 2) {
+            $aResponse['type'] = 'chimeric';
+        } else {
+            $aResponse['type'] = '=';
+        }
+
+    } elseif (strpos($aVariant['type'], '>') !== false) {
+        $aResponse['type'] = 'subst';
+        
+    } elseif (substr($aVariant['type'], -1) == ']') {
+        $aResponse['type'] = 'repeat';
+        $aResponse['warnings']['WNOTSUPPORTED'] =
+            'Repeat variants are currently not supported for mapping and validation.';
+        
+    } elseif ($aVariant['type'][0] == '|') {
+        $aResponse['type'] = 'met';
+
+    } elseif ($aVariant['type'] == '?') {
+        $aResponse['type'] = NULL;
+
+    } else {
+        $aResponse['type'] = $aVariant['type'];
+    }
+
 
     // If given, check if we already know this transcript.
     if ($sTranscriptID && !isset($aTranscriptOffsets[$sTranscriptID])) {
         $aTranscriptOffsets[$sTranscriptID] = $_DB->query('SELECT position_c_cds_end FROM ' . TABLE_TRANSCRIPTS . ' WHERE (id = ? OR id_ncbi = ?)',
             array($sTranscriptID, $sTranscriptID))->fetchColumn();
         if (!$aTranscriptOffsets[$sTranscriptID]) {
-            // Transcript not configured correctly.
-            // Don't die here; we might not even need these positions. We'll die later if we do.
+            // The transcript is not configured correctly. We will treat this transcript as unknown.
             $sTranscriptID = '';
         }
-    } elseif ($sTranscriptID === false) {
+
+    } elseif ($sTranscriptID == false) {
```

Comment:
Met deze verandering vervalt het verschil tussen `''` en `false`. Wat is de reden daar voor?

Reply:
Has been fixed.